### DISCUSSION TITLE
Multiple source directories - tag per dir?

### DISCUSSION THREAD

**Discussion by psyciknz** [djmaze/resticker#119] [Q&A]
Trying to get my head around various restic methods and containers.

I’ve got your container runnings and backing up all the container data I care about.
Just trying to see if I can add in other dirs on the host via mounts.   It seems to work, and they all appear in my /mnt/volumes directory.

but wondering if I can get a tag per source dir, or if that means I would run a new container for each other dir I add in to the volume statement.

or do I just handle these externally from the containers entirely?

Many questions.

**Comment by djmaze** [comment]
@psyciknz  Your question rather makes sense as a discussion, so I moved it here.

I am not quite sure what you mean with "a take per source dir". Could you elaborate?

**Reply by djmaze** [reply]
Ah, just saw the heading -_-. Okay got it. *tags*

Yeah, if you need different tags, then would need to run multiple containers I guess. I personally don't need different tags on a single host since I think it makes more sense to have tags on a host basis. So for me the current approach fits already.

**Reply by djmaze** [reply]
Also, restic itself does not allow different tags per directories as they are set *per backup*. So you need to do different backups either way in order to achieve what you want.

**Reply by psyciknz** [reply]
Yeah sorry was written on a iPad that has a habit of adding silly characters.

Am I being a pain by trying to use this container for backing up directories from the host of docker?

I like using the container as it gives me an easy way of holding Configuration and credentials for a repository over just doing it via scripts.

**Reply by djmaze** [reply]
Of course, backing up directories of the docker host is how it is supposed to be used normally (and as I am using it myself as well). Only having different tags per source directory is not supported by Restic.

**Reply by psyciknz** [reply]
I think I might just spin up others for the other source directories.

Or call them from externaly via docker-compose command

**Reply by djmaze** [reply]
Yeah, that's also possible.

In theory we could allow defining several backup jobs per container, but I think most people do not need this and it is rather a special requirement which might overly complicate the script.

**Reply by psyciknz** [reply]
Thanks for the quick replies.  I'll continue exploring. 

I'm embarking on a windows to Linux migration. And restic seems to be the backup tool of choice

**Reply by djmaze** [reply]
Congrats for the good choice(s) :)
